ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -68,7 +68,7 @@ jobs:
       class: RunPytest
       args:
         build_url: '{fedora-latest/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_commands.py::TestIPACommand::test_client_doesnot_throw_responsenotready_error
```

Comment:
In temp_commit please use only test_integration/test_commands.py::TestIPACommand